composer.json

```diff
@@ -14,5 +16,8 @@
         "psr-0": {
             "CareerBuilder\\": "src/"
         }
-    }
+    },
+    "repositories": [
+        { "type": "vcs",    "url": "git@github.com:cbdr/json-manager.git" }
```

Comment:
Don't inject a new dependency... Just override PHP's function and add error throwing like stated [here](https://github.com/cbdr/candidate-search-api/pull/38#discussion_r365334634).